libcxx/test/std/time/time.zone/time.zone.timezone/time.zone.members/get_info.local_time.pass.cpp

```diff
@@ -8,11 +8,10 @@
 
 // UNSUPPORTED: c++03, c++11, c++14, c++17
 // UNSUPPORTED: no-filesystem, no-localization, no-tzdb
+// REQUIRES: long_tests
```

Comment:
The test `libcxx/test/std/time/time.zone/time.zone.timezone/time.zone.members/sys_info.zdump.pass.cpp` has similar performance characteristics, can you mark that as a long test too?

Reply:
Yes please, I see that in the top of longest tests even without sanitizers.